qute-web-markdown/runtime/src/main/java/io/quarkiverse/qute/web/markdown/runtime/MarkdownSectionHelper.java

```diff
@@ -0,0 +1,57 @@
+package io.quarkiverse.qute.web.markdown.runtime;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Consumer;
+
+import io.quarkus.qute.CompletedStage;
+import io.quarkus.qute.EngineConfiguration;
+import io.quarkus.qute.ResultNode;
+import io.quarkus.qute.SectionHelper;
+import io.quarkus.qute.SectionHelperFactory;
+
+@EngineConfiguration
```

Comment:
If you place this annotation on the `Factory` class then the `QuteMarkdownConfigurator` is superfluous.

If you do that, then the `Factory` class has to declare a no-args constructor (passing the `CommonMarkConverter`?).